src/rgw/rgw_rest_s3.cc

```diff
@@ -6195,8 +6271,11 @@ void aws_response_handler::send_success_response()
     sql_result.append(END_PAYLOAD_LINE);
     int buff_len = create_message(header_size);
```

Comment:
what does the strlen(PAYLOAD_LINE) actually mean?  it can't be indicating that there is/isn't more data, can it?

Reply:
yes, it's not in place, probably should be removed (should test that).